ftw/book/content/book.py

```diff
@@ -84,12 +84,12 @@
 
                 widget=atapi.BooleanWidget(
                     label=_(u'book_label_use_index',
-                            default=u'Embedd subject index')),
-                description=_(u'book_help_use_index',
-                              default=u'When enabled, a keyword index '
-                              'will be included in the PDF.')),
+                            default=u'Embedd subject index'),
+                    description=_(u'book_help_use_index',
+                                  default=u'When enabled, a keyword index '
+                                           'will be included in the PDF.'))),
```

Comment:
Ooop, you were too fast ;-) Now my PEP 8 checker complains about over-indentation 🙈 So sorry.

Reply:
Shall I change it back? I don't know what our rule is in this case.

Reply:
I changed it to the way it was (starting at the same point as the u from unicode string starts).